utils/api/postcodeAPI.spec.ts

```diff
@@ -1,10 +1,15 @@
-import { normalizeAddress } from './postcodeAPI';
+import { formatAddress, lookupPostcode } from 'utils/api/postcodeAPI';
+import axios from 'axios';
+import { addressAPIWrapperFactory } from 'factories/postcode';
+
+jest.mock('axios');
+const mockedAxios = axios as jest.Mocked<typeof axios>;
 
 describe('postcodeAPI', () => {
   describe('normalizeAddresses', () => {
```

Comment:
```suggestion
  describe('formatAddress', () => {
```

Since we've renamed the function.